templates/about/volunteering.html

```diff
@@ -7,35 +7,35 @@
-</ul>
-<p>Please remember that everyone is a volunteer so you may not get an immediate response. Some teams are a lot more popular than others so may not need your assistance, but many others will be grateful for your help.</p>
+
 <h3 id="how-to-volunteer">How volunteering works</h3>
 <p>We understand that everyone who helps with EMF has other commitments and we plan ahead to cope with this.</p>
 <p>We have two golden rules for you:</p>
 <ul>
-<li>Don’t over-commit yourself</li>
-<li>Tell us if you can’t get something done</li>
+  <li>Don’t over-commit yourself</li>
+  <li>Tell us if you can’t get something done</li>
 </ul>
 <p>Nobody will criticise you if you can't do something, and telling us quickly means we can handle the situation.</p>
+
 <h3 id="volunteering-before-the-event">Volunteering before the event</h3>
 <p>Running an event like EMF requires a huge amount of planning to make sure the event runs smoothly. This starts a year or more in advance.</p>
+<p>We're looking for a small number of people to take responsibility for planning aspects of the event, and to help with design. If this is something that interests you take a look at the list of roles we're looking to fill.</p>
+<div class="list-group">
+  <a href="/about/volunteer-roles" class="list-group-item">Open Roles</a>
+</div>
+
 <h3 id="helping-out-during-the-event">Helping out during the event</h3>
 <p>During the event, there are plenty of roles which need to be filled, from construction to cooking for volunteers.</p>
 <p>Closer to the event, you’ll be able to log in to the volunteer system which allows you to sign up for shifts. Volunteers who complete a shift will receive a delicious free meal from our volunteer kitchen.</p>
+
 <h3 id="helping-with-setup-teardown">Helping with setup &amp; teardown</h3>
 <p>We’re grateful for help setting up the event, and especially tearing it down.</p>
 <p>If you want to help with either setup or teardown you will need a ticket to EMF and <u>permission from us to turn up early or stay after the event ends</u>. We do not allow under-18's on site during setup or teardown without explicit permission.</p>
 <p>During setup and teardown, EMF is a construction site and we are liable for your safety. If you don’t follow instructions we will be forced to remove you from site, for everyone's safety.</p>
 <p>If you’re on site for setup/teardown you have to help with everything - not just what you’re interested in. There’s plenty of lifting and shifting that requiring a degree of physical fitness and mobility. If you’re not actively helping, we will ask you to leave.</p>
-<small id="footnote">* We provide free tickets for qualified First Aid staff who must work a minimum of three eight-hour shifts during the event.</small></p>
+
+<p><small id="footnote">* We provide free tickets for qualified First Aid staff who must work a minimum of three eight-hour shifts during the event.</small></p>
```

Comment:
This might be better at the end of the section rather than the bottom of the page